lacework/provider.go

```diff
@@ -94,6 +94,7 @@ func Provider() *schema.Provider {
 			"lacework_resource_group_aws":            resourceLaceworkResourceGroupAws(),
 			"lacework_resource_group_azure":          resourceLaceworkResourceGroupAzure(),
 			"lacework_resource_group_gcp":            resourceLaceworkResourceGroupGcp(),
+			"lacework_resource_group_lw_account":     resourceLaceworkResourceGroupLwAccount(),
```

Comment:
I don't think we should add the `lw` into the resource name, could we update it to:
```suggestion
			"lacework_resource_group_account":     resourceLaceworkResourceGroupLwAccount(),
```

Reply:
I think the word account is very generic, especially in the context of resource groups.